doc/news/changes/minor/20230425Ivannikov

```diff
@@ -0,0 +1,6 @@
+Improved: Class DataOut_DoFData attempts now to automatically merge 
+scalar output components into vectors and tensors if the corresponding
+components are located next to each other and have the same name. 
+This is particularly relevant for scalar DoFHandler's.
```

Comment:
I would just remove this last sentence. I get what case you have in mind, but I suspect few will understand what "scalar DoFHandler" is supposed to mean :-)